script/Ciel/Day 10/10_01_CIEL10_3_1.txt

```diff
@@ -47,103 +50,112 @@ I have to value the classes I can attend, considering that I often miss them bec
 -- 鞄を手に、人気のない部活棟に足を向けた。
-I take my bag and make my way to the deserted wing of the school where the club rooms are.
+I grab my bag and make my way to the deserted wing of the school where the club rooms are located.
 }
 [sha:f32f16e9427aa81c8c6839146d65d6617595ced3]{
 -- Page 151, Offset 27466.
 -- 「あれ、志貴クン？　シエルさんと一緒じゃないの？」
-"Oh, Shiki-kun? Is Ciel-san not with you?"
+"Huh? Shiki-kun?
+ I thought you were with Ciel-san?"
 }
 [sha:3cb1542048ff64b01f55c822ac9226378308fb75]{
 -- Page 151, Offset 27467.
 -- 「ノエル先生こそ、どうして茶道室に？」
-"Noel-sensei, why are you in the tea ceremony room?"
+"I'm as surprised as you are, Noel-sensei.
+ What are you doing in the tea ceremony room?"
 }
 [sha:6735bc743ccbb56ba20408885406b9c85243fedb]{
 -- Page 152, Offset 27468.
 -- ……って、そうか。
-Oh... Wait.
+Oh... Duh.
 }
 [sha:65a7ab1a72bec8ac07e9ec465508ba18a44f6da7]{
 -- Page 152, Offset 27469.
 -- ここがシエル先輩の隠れ家なら、ノエル先生が利用していてもおかしくはない。
-This is Ciel-senpai's hideout, so there's no surprise to Noel-sensei also using it.
+This is Ciel-senpai's hideout,
+ so it shouldn't come as a shock to find Noel-sensei using it too.
 }
 [sha:f62f0163a6df392c7885f29d82b874b07c34bb6d]{
 -- Page 152, Offset 27470.
 -- 二人はペアの代行者なんだから。
-The two are working together as Executors after all.
+They're partners, after all. Executors always work in pairs.
 }
 [sha:2489477d588046252b5a2d1fea1ac9d535fc375e]{
 -- Page 153, Offset 27471.
 -- 「わたしはお留守番。お昼にちょっと、わたしたちの上の方で動きがあってね？
-"I'm just keeping a watch. There's been some development higher-up during lunch, right?
+"I'm just keeping watch.
+ See, our superiors started making a fuss around lunchtime, right?
 }
 [sha:301350e99bb1792db14f2bb7c357cd37c3e9bb5a]{
 -- Page 153, Offset 27472.
 -- マーリオゥってナマイキな司祭がいるんだけど、そいつにシエルさん呼ばれちゃったのよ。
-That cocky presbyter, you know, Mario? He summoned Ciel-san to him.
+There's this cocky presbyter called Mario, and he demanded Ciel-san go report to him at once.
 }
 [sha:8e559d983aaaa92b7db070b40dd30a6554409ddb]{
 -- Page 153, Offset 27473.
 -- “何か報告を忘れてねえか？”って。
-He was like 'Ya aren't forgettin' ya got a report to make?'
+He was like 'Ain'tcha forgetting to report somethin, huh?'
 }
 [sha:d77f569d78177d17e14ca760e827d43d1f65174a]{
 -- Page 153, Offset 27474. Mods: @k, @e.
 -- あー、もうイヤミでやんなっちゃう！
-Ahh, he gives me the creeps!
+%{i}Ugh%{/i}, he gives me the creeps!
 }
 [sha:0f603eb744ada1aecd9dba76d8ace725f9aa3d81]{
 -- Page 153, Offset 27475. Glued. Mods: @x.
 -- はい、志貴クンにもお茶とケーキ！　二人で一緒にヤケ喰いしちゃおうぜ！」
 // I love her so much
-Okay, there you go, tea and cake, Shiki-kun! It's binge eating time, just the two of us!"
+ Here, you have some tea and cake too, Shiki-kun!
+ It's binge eating time, just the two of us!"
 }
 [sha:72b8e809831744779fe57d380785d146d3a1fa9c]{
 -- Page 154, Offset 27476.
 -- 和室で寛ぐノエル先生の前には、お茶とショートケーキのセットがあった。
-Noel-sensei is relaxing in the traditional Japanese room in front of a set of tea and shortcake.
+Noel-sensei is relaxing in the traditional Japanese room in front of a tray loaded with tea and shortcake.
```

Comment:
I mean if you're going to drop the 和 in 和室 you may as well just snip that clause altogether, it doesn't really add anything anymore. Maybe just reference the tatami instead of floor for jp spite.
> There's a tray loaded with tea and shortcake laid out on the tatami in front of Noel-sensei.

Reply:
Yeah, that was a consideration. It's probably the wisest move.